tests/test_generate_url.py

```diff
@@ -15,12 +15,15 @@ def setUp(self) -> None:
     def test_generate_url_with_path(self):
         options = {
             "path": "/default-image.jpg",
-            "transformation": [{"height": "300", "width": "400"}],
+            "transformation": [{"raw":"f-auto","height": "300", "width": "400"}],
```

Comment:
Please avoid modifying existing test case. Instead add a new test case just to assert this.